CMakeLists.txt

```diff
@@ -473,6 +465,33 @@ else()
     find_package(GDAL)
   endif()
 
+  FIND_PACKAGE(Lua 5.2 EXACT)
```

Comment:
I think we can now simplify all this logic down to `find_package(Lua 5.1)` which will look for lua 5.1+. Sol seems to have a good support for all lua versions, unlike luabind.